governance/votes/Executive vote - July 11, 2023.md

```diff
@@ -0,0 +1,122 @@
+date: $2023-07-11T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Stability Scope Parameter Changes, Smart Burn Engine Deployment, DssCron Housekeeping, Various Budget Streams and Payments - July 11, 2023
+
+The $exec_publishers and $exec_crafters have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Smart Burn Engine will be deployed and the old MCD_Flapper will be disabled
+- The Dai Savings Rate will be reduced from 3.49% to 3.19%
+- Multiple risk parameters will be changed including Stability Fees, Maximum Debt Ceilings and Liquidation Ratios, as detailed below.
+- DssCron Housekeeping Additions
+
+If this executive proposal passes, the following **Actions** will occur within Maker Protocol:
+- Aligned Delegate One-Off Compensation totaling **X MKR** will be distributed
+- 125 MKR will be transfered to DECO-001
+- DAI and MKR streams will be set up for Chronicle Labs, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Smart Burn Engine Deployment
+
+As per [this poll](https://vote.makerdao.com/polling/QmQmxEZp), the Smart Burn Engine will be implemented and the old flapper Depreciated. The following actions will occur
+
+- Depreciate MCD_FLAP
+- Deploy [FlapperUniV2](https://github.com/makerdao/dss-flappers/blob/univ2/README.md)
+- Reduce vow.hump by 200 million DAI from 250 million DAI to 50 million DAI
+- Reduce vow.bump by 25,000 DAI from 30,000 DAI to 5,000 DAI
+- Set hop to 1,577 seconds
+- Set want to 0.98
+- Set receiver to MCD_PAUSE_PROXY
+- Set pip to TBD
+
+### Stability Scope Parameter Changes
+
+As per the Stability Scope Facilitator, SES, [the following changes](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238) will be implemented. 
```

Comment:
Nope. This is unclear. SES aren't the reason these changes are happening, they are the authority permitting them to happen due to the facilitator powers, but they are not the origin point. 

As a general rule, things are included in these proposals because governance made a decision at some point. In this case, the ratification of the stability scope. 

So the changes are justified by the ratification of the stability scope. The addition of the ratified changes to this executive proposal were authorized by SES as the Stability Scope Facilitators.

So, links here would be:
* Link to the ratified version of the Stability Scope 'as per the stability scope...'
* Link to the post from the Stability Scope Facilitators '...and as authorized by the Stability Scope Facilitators...
* Link to the changes themselves in the top level forum post. '...the following changes...' 

And as above: 'will be implemented _if this executive proposal passes_'